### PR TITLE
7764 - Added PaySystemAlert to namerequest

### PR DESCRIPTION
**Pull request by andrepestana-aot**
*Issue #:* /bcgov/entity#7764

*Description of changes:*

- Added PaySystemAlert to namerequest

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namerequest license (Apache 2.0).

### CODE REVIEW

File: src/plugins/vuetify.ts [hunk 1]
```diff
@@ -21,7 +21,7 @@ export function getVuetify () {
           error: '#d3272c', // same as $app-red
           info: '#2196F3',
           success: '#1a9031', // same as $app-green
-          warning: '#FFC107'
+          warning: '#FF9800'
```
Comment (severinbeauvais) [inline src/plugins/vuetify.ts:24]: Is this used anywhere else? (eg, button colors)

Comment (andrepestana-aot) [reply src/plugins/vuetify.ts:24]: No, the only warning is in App.vue

**Comment by andrepestana-aot** [comment]
Changed to draft as UX wants it yellow as it was with black text.

**Review by severinbeauvais — COMMENTED** [review]
Looks good. Let's let someone else approve it this time :)

**Review by cameron-freshworks — APPROVED** [review]
Looks good!

**Pull request merged**